src/components/createEventModal.vue

```diff
@@ -1,27 +1,31 @@
 <template>
-<transition name="fade" mode="out-in">
-  <div class="modal-backdrop">
-    <div class="nav-icons">
-      <router-link to="/discover">
-        <font-awesome-icon icon="arrow-left" class="back-btn fa-2x" color="#1F2845"/>
-      </router-link>
-    </div>
-    <div class="modal-container">
-      <div class="input-box">
-        <div class="input-wrapper">
-          <v-form class="input-form">
-            <v-text-field
-              name="event-title"
-              label="Event Title"
-              class="event-info-input"
-              clearable
-              required
-            ></v-text-field>
-          </v-form>
-        </div>
+  <transition name="fade" mode="out-in">
```

Comment:
Idk but I think the modal needs a bunch more refactoring (in my head I'm thinking plain solid white background instead)

Reply:
let's remove this modal entirely since we have the event initiation process

Reply:
also this modal shouldn't be at all visible on mobile - which is again what we're focusing on for now
**this doesn't mean we shouldn't make components responsive for desktop**

Reply:
Let's completely remove it 👍

Reply:
If can remove the modal in this PR too